VS_Meadow_Extension/VS_Meadow_Extension.Shared/MeadowDeployProvider.cs

```diff
@@ -112,17 +114,18 @@ public async Task DeployAsync(CancellationToken cancellationToken, TextWriter te
                 var packageManager = new PackageManager(fileManager);
 
                 outputLogger.Log("Trimming application binaries...");
-
                 await packageManager.TrimApplication(new FileInfo(Path.Combine(outputPath, "App.dll")), osVersion, includePdbs, cancellationToken: cancellationToken);
 
-                await Task.Run(async () => await AppManager.DeployApplication(packageManager, connection, osVersion, outputPath, includePdbs, false, outputLogger, cancellationToken));
+                outputLogger.Log("Deploying application...");
+                await AppManager.DeployApplication(packageManager, connection, osVersion, outputPath, includePdbs, false, outputLogger, cancellationToken);
 
                 await connection.RuntimeEnable();
 
                 await outputLogger.ShowBuildOutputPane();
             }
             finally
             {
+                connection.FileWriteProgress -= MeadowConnection_DeploymentProgress;
```

Comment:
I think we want to keep this so we continue to get application output ... ? (I'll test)

Reply:
@adrianstevens this disconnect the file progress, the other one is still hooked up for application output.